aperturedb/Utils.py

```diff
@@ -14,6 +20,51 @@
 DEFAULT_METADATA_BATCH_SIZE = 100_000
 
 
+def __create_connector(configuration: Configuration):
+    if configuration.use_rest:
+        connector = ConnectorRest(
+            host=configuration.host,
+            port=configuration.port,
+            user=configuration.username,
+            password=configuration.password,
+            config=configuration)
+    else:
+        connector = Connector(
+            host=configuration.host,
+            port=configuration.port,
+            user=configuration.username,
+            password=configuration.password,
+            config=configuration)
+    console.log(f"Connected Using:")
+    console.log(configuration)
+    return connector
+
+
+def create_connector():
+    """
+    **Create a connector to the database.**
+
+    Args:
+        None
+
+    Returns:
+        Connector: The connector to the database.
+    """
+    all_configs = ls()
+
+    env_config = os.environ.get("APERTUREDB_CONFIG")
+    ac = all_configs["active"]
+    config = all_configs["local"][ac] if "local" in all_configs else all_configs["global"][ac]
+
+    if env_config is not None:
+        # TODO test me.
```

Comment:
I think this is tested, right?
```suggestion
```

Reply:
This means that the using active config by setting environment was not tested, nor has automated tests.